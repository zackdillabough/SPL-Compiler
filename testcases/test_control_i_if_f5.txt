
a , b ;
{
input a;
input b;
IF b > a {
        a = a+b;
	output a ;
	IF a > b { 
		output a ;
		b = a+b;
		IF b > a {
			output a;
			output b;
			IF a > b {
				output a;
				output b;
			}
		}
	}
	a = a+b;
	output a;
}
a = a*b;
output a;
output b;

}
13 25 32 0 9 2
