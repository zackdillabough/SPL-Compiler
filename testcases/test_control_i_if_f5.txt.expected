38 38 38 63 101 6363 63 